Composer/packages/lib/shared/src/skillsUtils/index.ts

```diff
@@ -149,11 +149,9 @@ export const parseRuntimeKey = (
 
 export const isManifestJson = (content) => {
   try {
-    const urls = [
-      'https://schemas.botframework.com/schemas/skills/v2.1/skill-manifest.json',
-      'https://schemas.botframework.com/schemas/skills/v2.2/skill-manifest.json',
-    ];
-    return urls.includes(JSON.parse(content).$schema);
+    const versions = ['/v2.0/', '/v2.1/', '/v2.2/'];
```

Comment:
Is there any reason we should worry about a `$schema` field matching these shorter version strings but being something other than a longer URL (like the ones that used to be here)?

Reply:
While the entire path is more correct, this seems less restrictive. I had a mistake in a manifest schema path, while the rest of the manifest was correct. I do not feel strongly about this though. If you object, we can skip this change.

Maybe we should be checking the $version, instead of the #schema path.  Considering, our previous .schema had no version in the $id path: https://schemas.botframework.com/schemas/skills/v2.0/skill-manifest.json 

```
"$id": "https://schemas.botframework.com/schemas/skills/skill-manifest.json",
```

Reply:
@beyackle adjusted to endsWith '{v}/skill-manifest.json' and remove v2.0, considering the above comment